operations/mimir-mixin-compiled/alerts.yaml

```diff
@@ -17,13 +17,26 @@ groups:
         The route {{ $labels.route }} in {{ $labels.cluster }}/{{ $labels.namespace }} is experiencing {{ printf "%.2f" $value }}% errors.
       runbook_url: https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#mimirrequesterrors
     expr: |
-      100 * sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{status_code=~"5..",route!~"ready"}[1m]))
+      100 * sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{status_code=~"5..",route!~"ready|debug_pprof"}[1m]))
         /
-      sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{route!~"ready"}[1m]))
+      sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{route!~"ready|debug_pprof"}[1m]))
         > 1
     for: 15m
     labels:
       severity: critical
+  - alert: MimirRequestErrors
+    annotations:
+      message: |
+        The route {{ $labels.route }} in {{ $labels.cluster }}/{{ $labels.namespace }} is experiencing {{ printf "%.2f" $value }}% errors.
+      runbook_url: https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#mimirrequesterrors
+    expr: |
+      100 * sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{status_code=~"5..",route=~"debug_pprof"}[1m]))
+        /
+      sum by (cluster, namespace, job, route) (rate(cortex_request_duration_seconds_count{route=~"debug_pprof"}[1m]))
+        > 1
+    for: 15m
+    labels:
+      severity: warning
```

Comment:
Why would we want this alert?

Reply:
I agree with Peter. I don't think we want this alert at all, even if it's a warning.

Reply:
For me, it'd be weird if this endpoint was regularly failing, and I'd want to investigate the issue, but it's not enough to get someone out of bed for.

But I'm also happy just ignoring failures - I'll amend the PR to do just that.

Reply:
From Mimir's pov, this is an endpoint for debugging and troubleshooting. For it to fail regularly, it needs to be used from somewhere, like Grafana Pyroscope. It would expect it to be Pyroscope's job to complain when it can't fetch data.